.github/workflows/benchmark.yml

```
@@ -0,0 +1,70 @@
+name: Run benchmarks
+
+on:
+  push:
+    branches: [setup_action]
```

It should be changed to:

```suggestion
    branches: [main]
```